routes/oneroster/v1_0/academicsessions.js

```diff
@@ -56,17 +59,18 @@ function queryAcademicSession(req, res, next, type) {
             });
         }
     });
-};
+}
 
 function queryAcademicSessions(req, res, next, type) {
+
     db.getData(req, res, {
         table: table,
         queryValues: [type],
         additionalWhereStmts: type ? 'type = ?' : ''
-    }).then((data) => {
+    }, next).then((data) => {
```

Comment:
db.getData doesn't take a next argument.

https://github.com/CarnegieLearningWeb/uniroster-server/blob/d6b83e3b621aa22d38cc2e5bef0130d26bb88397/utils/database.js#L36

Reply:
Yeah, I don't know how I forgot that because I removed it, anyway, thanks. I've pushed the change.